src/Umbraco.Web.UI.Client/src/packages/tiptap/components/input-tiptap/input-tiptap.element.ts

```diff
@@ -138,6 +138,11 @@ export class UmbInputTiptapElement extends UmbFormControlMixin<string, typeof Um
 		});
 	}
 
+	#getLabel(suffix?: string) {
+		const base = this.label ? this.label + (this.required ? ' *' : '') : 'Rich Text Editor';
+		return suffix ? `${base} ${suffix}` : base;
+	}
+
```

Comment:
If we revert the unrequired `aria-label` attributes (on toolbar and statusbar), then we probably don't need this to be a separate method/function.

With the `required` conditional, that would be inserting an asterisk into the label's text, which I believe a screen-reader would literally read it out, without the association of it being required/mandatory.

I think we could add a `aria-required` attribute to the `editorProps` (line 181).

The other part we're missing here is that the "Rich Text Editor" could be localized. 🤔